armi/reactor/converters/axialExpansionChanger/expansionData.py

```diff
@@ -28,82 +28,103 @@
     Flags.SLUG,
 ]
 
+if TYPE_CHECKING:
+    from armi.reactor.components import Component
+    from armi.reactor.blocks import Block
+    from armi.reactor.assemblies import Assembly
 
-def getSolidComponents(b):
+
+def iterSolidComponents(b: "Block") -> Iterable["Component"]:
+    """Iterate over all solid components in the block."""
+    return filter(lambda c: not isinstance(c.material, material.Fluid), b)
```

Comment:
Tis fancier, yes. But is this better than looping approach that existed before? Maybe it's so it can be used in the filter function in `_findComponentLinkedTo`?

EDIT: I see your `see also` statement... more teaching!